### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by riccardocadei**
update CRAN package title and description

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -16,19 +16,15 @@ Authors@R: c(
...
-    features space into heterogeneous subgroups. A small number of
-    significant causal rules are selected through Stability Selection to
-    control for family-wise error rate in the finite sample setting. It
-    proposes various estimation methods for the conditional causal effects
-    for each discovered causal rule.  It is highly flexible and multiple
-    causal estimands and imputation methods are implemented.  Lee, K.,
-    Bargagli-Stoffi, F. J., & Dominici, F. (2020). Causal rule ensemble:
-    Interpretable inference of heterogeneous treatment effects.  arXiv
-    preprint <arXiv:2009.09036>.
+Description: Causal Rule Ensemble is a new method for interpretable
```
Comment (Naeemkh) [inline DESCRIPTION:19]: Please start with provides and drop Causal Rule Ensemble.

File: DESCRIPTION [hunk 2]
```diff
@@ -16,19 +16,15 @@ Authors@R: c(
...
-    causal estimands and imputation methods are implemented.  Lee, K.,
-    Bargagli-Stoffi, F. J., & Dominici, F. (2020). Causal rule ensemble:
-    Interpretable inference of heterogeneous treatment effects.  arXiv
-    preprint <arXiv:2009.09036>.
+Description: Causal Rule Ensemble is a new method for interpretable 
+    heterogeneous treatment effects characterization in terms of decision 
+    rules via an extensive exploration of heterogeneity patterns by an 
+    ensemble-of-trees approach, enforcing high stability in the discovery.
+    It relies on a two-stage pseudo-outcome regression, and it is
+    supported by theoretical convergence guarantees.
```
Comment (Naeemkh) [inline DESCRIPTION:24]: Reference should follow the description, drop the new line.

File: DESCRIPTION (hunk 1 above)
Comment (riccardocadei) [reply DESCRIPTION]: fixed

File: DESCRIPTION (hunk 2 above)
Comment (riccardocadei) [reply DESCRIPTION]: fixed

**Pull request merged**